moveit_planners/trajopt/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(moveit_planners_trajopt)
+
+# At least C++11 required for TrajOpt
+add_compile_options(-std=c++14)
+
+if(NOT CMAKE_CONFIGURATION_TYPES AND NOT CMAKE_BUILD_TYPE)
+  set(CMAKE_BUILD_TYPE Release)
+endif()
+
+find_package(Boost REQUIRED system filesystem date_time thread serialization)
+
+find_package(catkin REQUIRED COMPONENTS
+  moveit_core
+  moveit_ros_planning
+  roscpp
+  pluginlib
+)
+
+catkin_package(
+  CATKIN_DEPENDS
+    moveit_core
+)
+
+# the following include_directory should have include folder of the new planner. For example for ompl, we need to add "ompl_interface/include"
```

Comment:
line too long
remove reference to ompl
capitalize first word :-)

Reply:
corrected it.